IntegrationTests/FSTest.js

```diff
@@ -63,7 +63,7 @@ function testWriteAndReadFile() {
 
 
 function testCreateAndDeleteFile() {
-  var path = RNFS.DocumentDirectoryPath + '/test.txt';
+  var path = RNFS.ExternalDirectoryPath + '/test.txt';
```

Comment:
This test does not work for iOS does it? Because `ExternalDirectoryPath` is `null`.